Matrix/inc/Matrix.hpp

```diff
@@ -30,6 +30,7 @@ class Matrix
     double &operator()(int x, int y);
     friend bool operator==(const Matrix &lhs, const Matrix &rhs);
     Matrix &operator+(Matrix &mat_1);
+	Matrix &operator+=(const Matrix& mat1);
```

Comment:
please correct the identation

Reply:
I checked my second commit, and this still appears to be indented with the github viewer. On my machine, it looks correct though